### ISSUE TITLE
Not listing used actions for all the repos under an organisation

### ISSUE THREAD

**Issue by 1633605-Phaneendra** [devops-actions/load-used-actions#3]
I have used this action to list out all the actions being used across repos under a given organisation. JSON export and debug logs shows only the actions being used in current repository where the workflow is being executed. Also the scope of the token is having org level access and added organization parameter to the action.

**Comment by 1633605-Phaneendra** [comment]
@rajbos Request your feedback here.

**Comment by rajbos** [comment]
I will look into it. Not sure when.

**Comment by rajbos** [comment]
Can you check the logs of the run to see if this line is in there? Just to check if the organization parameter comes through correctly.

`Using default for organization: [devops-actions]`

I've cleaned up the logging and added a test job in the [testing workflow](https://github.com/devops-actions/load-used-actions/blob/main/.github/workflows/testing.yml) to shows this works with an organization. You can find the last run [here](https://github.com/devops-actions/load-used-actions/actions/runs/1578032200)

**Comment by 1633605-Phaneendra** [comment]
Here is the log from the workflow.

We're running with these parameters:
 PAT.Length: [40]
 orgName: [scb-et]
 userName: [1633605-Phaneendra]
 marketplaceRepo: []
Calling api on url [https://api.github.com/orgs/scb-et/repos]

Does it support only public repos or also supports internal and private kind of repos ?

**Comment by rajbos** [comment]
What are you using against the PAT parameter? 

If you use the [GITHUB_TOKEN](https://docs.github.com/en/actions/security-guides/automatic-token-authentication), it will only show the public repositories for the organization if any. If you give it a Personal Access Token or from a GitHub App that has been installed in the repositories you want to analyze, this will return these repositories for you. 

I verified this with a private repo in this org and it picks up the workflows if you use a PAT, but not with a GITHUB_TOKEN. I updated the readme with that information.
